src/smb_ntlm.c

```diff
@@ -382,9 +381,15 @@ void        smb_ntlmssp_response(uint64_t srv_challenge, uint64_t srv_ts,
     auth->flags = 0x60088215;
 
 
-    __AUTH_APPEND(lm, lm2, 24, cursor)
-    __AUTH_APPEND(ntlm, ntlm2, blob_size + 16, cursor)
-
+    if (!*domain && !*user && !*password) {
+        utf_sz = smb_to_utf16("", 0, &utf);
```

Comment:
yes, you are right. actually, there are no need to do the hash function to get smb_lm2_response and smb_ntlm2_response. should I remove it ?